charts/ocis/values.yaml

```diff
@@ -2295,209 +5072,504 @@ services:
     # Configuration for web, that ends up in the config.json file.
     config:
       tokenStorageLocal:
+        # @schema
+        # type: boolean
+        # required: true
+        # @schema
         #-- Specifies whether the access token will be stored in the local storage when set to ’true’ or in the session storage when set to ‘false’.
         # If stored in the local storage, login state will be persisted across multiple browser tabs, means no additional logins are required.
         enabled: true
       contextHelpersReadMore:
+        # @schema
+        # type: boolean
+        # required: true
+        # @schema
         # -- Specifies whether the “Read more” link should be displayed or not.
         enabled: true
       feedbackLink:
+        # @schema
+        # type: boolean
+        # required: true
+        # @schema
         # -- Enables the feedback link in the Web UI.
         enabled: true
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
         # -- URI where the feedback link points. Uses the ownCloud default href if empty.
-        href: ""
+        href:
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
         # -- Screen reader accessible label for the feedback link. Uses the ownCloud default label if empty.
-        ariaLabel: ""
+        ariaLabel:
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
         # -- Description to be shown for the feedback link. Uses the ownCloud default description if empty.
-        description: ""
+        description:
+      # @schema
+      # additionalProperties: true
+      # @schema
       # -- Configure the {"styles": []} section in the Web config.json.
       styles:
         []
         # - href: /theme/foo.css
+      # @schema
+      # additionalProperties: true
+      # @schema
       # -- Configure the {"styles": []} section in the Web config.json.
```

Comment:
Seems to me that this description of the setting is wrong.

Reply:
fixed